### ISSUE TITLE
Relationship lines in ERDs are difficult to follow (RM #7198)

### ISSUE THREAD

**Issue by dpage** [pgadmin-org/pgadmin4#4915]
***Issue migrated from Redmine: https://redmine.postgresql.org/issues/7198***
*Originally created by **David P-C** at 2022-02-22 20:33:17 UTC.*

The diagrams generated by the ERD Tool in pgAdmin are often confusing because the relationship lines always extend from the right side of tables, and often pass behind other tables. The diagrams could be greatly improved by allowing lines to extend from either side of a table and routing lines around other objects rather than behind them. See the attached image for an example of a diagram with hard to follow lines. Look at DbVisualizer or other applications that generate ERDs to see examples of better handling of relationship lines.

**Comment by adityatoshniwal** [comment]
Duplicates #6592.
